client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -75,11 +83,14 @@ export default class ReserveSubmitButton extends Vue {
       this.sendToExamination()
       return
     }
-    if (['add_descriptive', 'add_distinctive'].includes(newReqModule.currentIssue.issue_type)) {
-      if (!newReqModule.showActualInput) {
-        this.$root.$emit('show-original-name')
+    if (newReqModule.currentIssue && newReqModule.currentIssue.issue_type) {
```

Comment:
If the TS version of NameRequest is at least 3.7, you can probably get away with optional chaining here. 
ie `newReqModule.currentIssue?.issue_type`. This would prevent the error `issue_type` doesn't exist on null/undefined etc.

Reply:
The package.json says ^3.5.3 ... which got auto-updated to 3.9.3 👍

Reply:
if that works the way i think it should (ie. preventing pyramid of doom scenario), then i cannot believe i didn't know about this.

Reply:
oh ya, pretty new to me as well! I think it was added roughly this time last year.

Reply:
Haha, totally Scott!
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
TypeScript is now closer than ever to C# (not surprising since they're both from MS). But I like it!